keyboards/keebio/bfo9000/keymaps/insertsnideremarks/keymap.c

```diff
@@ -0,0 +1,386 @@
+  |       |       |       |       |       |       |       |       |       |          |       |       |       |       |       |       |       |       |       |
+  |-------+-------+-------+-------+-------+-------+-------+-------+-------|          |-------+-------+-------+-------+-------+-------+-------+-------+-------|
+  |       |       |       |       |       |       |       |       |       |          |       |       |       |       |       |       |       |       |       |
+  `-----------------------------------------------------------------------'          `-----------------------------------------------------------------------'
+*/
+[_ADJUST] = LAYOUT(
+  _______, _______, _______, _______, _______,   _______,  _______, _______, _______,     _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, COLEMAK, QWERTY,  _______, COLEMAKGM, QWERTYGM, _______, _______, _______,     _______, _______, _______, NUMPAD,  _______, _______, _______, _______, RESET,
+  _______, _______, _______, _______, _______,   _______,  _______, _______, _______,     _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______,   _______,  _______, _______, _______,     _______, _______, _______, _______, NKROTG,  _______, _______, _______, _______,
+  _______, _______, _______, _______, _______,   _______,  _______, _______, _______,     _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______,   _______,  _______, _______, _______,     _______, _______, _______, _______, _______, _______, _______, _______, _______
+),
+
+[_ADJUST2] = LAYOUT(
+  _______, _______, _______, _______, _______,   _______,  _______, _______, _______,     _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, COLEMAK, QWERTY,  _______, COLEMAKGM, QWERTYGM, _______, _______, _______,     _______, _______, _______, NUMPAD,  _______, _______, _______, _______, RESET,
+  _______, _______, _______, _______, _______,   _______,  _______, _______, _______,     _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______,   _______,  _______, _______, _______,     _______, _______, _______, _______, NKROTG,  _______, _______, _______, _______,
+  _______, _______, _______, _______, _______,   _______,  _______, _______, _______,     _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______,   _______,  _______, _______, _______,     _______, _______, _______, _______, _______, _______, _______, _______, _______
+)
+
+};
+
+
+uint32_t layer_state_set_user(uint32_t state) {
+  return update_tri_layer_state(state, _NUMBERS, _NUMBERS2, _ADJUST);
+}
+
+void persistent_default_layer_set(uint16_t default_layer) {
```

Comment:
Could you remove this function?  It's not needed.

Reply:
Will do - pretty sure I got them removed from all three keymaps.